src/plugins/htmlAudioPlayer/plugin.js

```diff
@@ -1,91 +1,94 @@
-            cancelFadeTimeout();
-            fadeTimeout = setTimeout(function () {
-                fade(instance, elem, newVolume).then(resolve, reject);
-            }, 100);
-        });
+    return new Promise(function (resolve, reject) {
+        cancelFadeTimeout();
+        fadeTimeout = setTimeout(function () {
+            fade(instance, elem, newVolume).then(resolve, reject);
+        }, 100);
+    });
+}
+
+function cancelFadeTimeout() {
+    const timeout = fadeTimeout;
+    if (timeout) {
+        clearTimeout(timeout);
+        fadeTimeout = null;
     }
+}
 
-    function cancelFadeTimeout() {
-        var timeout = fadeTimeout;
-        if (timeout) {
-            clearTimeout(timeout);
-            fadeTimeout = null;
-        }
+function supportsFade() {
+    if (browser.tv) {
+        // Not working on tizen.
+        // We could possibly enable on other tv's, but all smart tv browsers tend to be pretty primitive
+        return false;
     }
 
-    function supportsFade() {
-        if (browser.tv) {
-            // Not working on tizen.
-            // We could possibly enable on other tv's, but all smart tv browsers tend to be pretty primitive
-            return false;
-        }
+    return true;
+}
 
-        return true;
-    }
+function requireHlsPlayer(callback) {
+    import('hlsjs').then(({default: hls}) => {
+        window.Hls = hls;
+        callback();
+    });
+}
 
-    function requireHlsPlayer(callback) {
-        require(['hlsjs'], function (hls) {
-            window.Hls = hls;
-            callback();
-        });
+function enableHlsPlayer(url, item, mediaSource, mediaType) {
+    if (!htmlMediaHelper.enableHlsJsPlayer(mediaSource.RunTimeTicks, mediaType)) {
+        return Promise.reject();
     }
 
-    function enableHlsPlayer(url, item, mediaSource, mediaType) {
-        if (!htmlMediaHelper.enableHlsJsPlayer(mediaSource.RunTimeTicks, mediaType)) {
-            return Promise.reject();
-        }
-
-        if (url.indexOf('.m3u8') !== -1) {
-            return Promise.resolve();
-        }
+    if (url.indexOf('.m3u8') !== -1) {
+        return Promise.resolve();
+    }
 
-        // issue head request to get content type
-        return new Promise(function (resolve, reject) {
-            require(['fetchHelper'], function (fetchHelper) {
-                fetchHelper.ajax({
-                    url: url,
-                    type: 'HEAD'
-                }).then(function (response) {
-                    var contentType = (response.headers.get('Content-Type') || '').toLowerCase();
-                    if (contentType === 'application/x-mpegurl') {
-                        resolve();
-                    } else {
-                        reject();
-                    }
-                }, reject);
-            });
+    // issue head request to get content type
+    return new Promise(function (resolve, reject) {
+        import('fetchHelper').then(({default: fetchHelper}) => {
```

Comment:
`fetchHelper` doesn't have `default` export.
```suggestion
        import('fetchHelper').then((fetchHelper) => {
```
Or we must add `default` export.

Reply:
Fixed